### PR TITLE
Fix roles fragment issue

### PR DESCRIPTION
**Pull request by ojasgulati**
Please make sure these boxes are checked before submitting your pull request - thanks!

Fixes #{Issue Number 171}

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: app/src/main/java/org/apache/fineract/ui/online/roles/roleslist/RolesPresenter.java [hunk 1]
```diff
@@ -37,6 +37,12 @@ protected RolesPresenter(@ApplicationContext Context context,
         compositeDisposable = new CompositeDisposable();
     }
 
+    @Override
+    public void detachView() {
+        super.detachView();
+        compositeDisposable.clear();
+    }
+
```
Comment (therajanmaurya) [inline app/src/main/java/org/apache/fineract/ui/online/roles/roleslist/RolesPresenter.java]: Please override the `attachView` too.

Comment (ojasgulati) [reply app/src/main/java/org/apache/fineract/ui/online/roles/roleslist/RolesPresenter.java]: changes done! Please have a look :)

**Comment by therajanmaurya** [comment]
@ojasgulati Please follow the commit guideline: https://github.com/openMF/mifos-mobile/wiki/Commit-Style-Guide and force push again.

**Comment by ojasgulati** [comment]
All done! Will follow the guidelines in future :)

**Pull request merged**